lib/block/blockheader.js

```diff
@@ -212,6 +217,22 @@ BlockHeader.prototype.getTargetDifficulty = function getTargetDifficulty(info) {
 };
 
 /**
+ * @link https://en.bitcoin.it/wiki/Difficulty
+ * @param {Number} genesisBits
```

Comment:
It should noted that this is optional, and will default to the bits of livenet and testnet genesis bits.